docs/doc_sources/beginners_guides/misc.rst

```diff
@@ -0,0 +1,69 @@
+.. _beginners_guide_why_dpctl:
+
+History of ``dpctl`` name
```

Comment:
I wonder if this page should either be renamed or moved to the beginner's guides index page in the future. It stands out when looking at the list.